proposals/0292-package-registry-service.md

```diff
@@ -471,8 +466,8 @@ $ cat .swiftpm/config/registries.json
 ```json
 {
   "registries": {
```

Comment:
should `registries` be an array?

Reply:
I don't think it should, no.

As we discussed before, the use of an array for a ranked list of "fallback" registries has negative security and usability implications. Our design uses scoped registries, which offer the flexibility to override a default registry without creating ambiguity about where a package is coming from.